backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -153,7 +155,48 @@ private Specification<TestEvent> buildTestEventSearchFilter(
       return p;
     };
   }
+  // methods to replace the methods that return List<TestEvent> once we get
+  // backwards compability issues resolved
+  @Transactional(readOnly = true)
+  @AuthorizationConfiguration.RequirePermissionReadResultListAtFacility
+  public Page<TestEvent> getTestEventsResultsPage(
+      UUID facilityId,
+      UUID patientId,
+      TestResult result,
+      PersonRole role,
+      Date startDate,
+      Date endDate,
+      int pageOffset,
+      int pageSize) {
+
+    PageRequest pageRequest =
+        PageRequest.of(pageOffset, pageSize, Sort.by("createdAt").descending());
```

Comment:
Out of scope for this PR, but I wonder if we should adjust this to "updatedAt" instead of "createdAt". Thinking specifically of edited tests, which can sometimes be hard to find after you've edited them. (This of course leads to the broader question of including any kind of "submitted at" column on the frontend...but that's also out of scope)